src/style.css

```
@@ -0,0 +1,78 @@
+[data-qa="big"] {
+  font-family: Avenir, Book, sans-serif;
+  margin-top: 12px;
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 24px ;
+  line-height: 22px;
+  background-size: 19px;
+  padding-left: 62px;
+  font-weight: 400;
```

```suggestion
  padding-left: 62px;
```
resundant - 400 is default font-weight